code/modules/research/xenobiology/crossbreeding/_clothing.dm

```diff
@@ -117,6 +117,14 @@ Slimecrossing Armor
 			return
 	return ..()
 
+/obj/item/clothing/head/peaceflower/MouseDrop(atom/over, src_location, over_location, src_control, over_control, params)
+	if(iscarbon(usr))
```

Comment:
Instead of copying and pasting the code, move it to its own proc that it and `attack_hand` both call.